client/app/components/Input/__tests__/InputSwitch.spec.jsx

```diff
@@ -35,7 +35,7 @@ describe('InputSwitch', () => {
       expect(screen.getByRole('checkbox')).not.toBeChecked();
 
       /**
-       * TODO: Follow up on `userEvent.type(inputSwitch, '{enter}')` in v12.1.7.
+       * TODO: Follow up on `await userEvent.type(inputSwitch, '{enter}')` in v12.1.7.
```

Comment:
Let's also make a GitHub issue for this, so it doesn't get lost in comments.

Reply:
This is an old comment that I was able to just resolve & remove as part of this upgrade 🎉 !